utf8/valid_go_test.go

```diff
@@ -0,0 +1,256 @@
+package utf8
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+	"testing"
+	stdutf8 "unicode/utf8"
+
+	stdascii "github.com/segmentio/asm/ascii"
```

Comment:
Should this be ascii? It's a bit misleading to call it std as we also used stdutf8 for the stdlib utf8 package.